packages/indexer-agent/src/agent.ts

```diff
@@ -1010,6 +1041,7 @@ class Agent {
       epochStartBlock,
       this.indexer.indexerAddress,
     )
+    const tempPoi = await this.resolvePOI(epochStartBlock, existingAllocation)
```

Comment:
I think it's worth reaching out a bit to the indexer community to decide on how this should be handled, but I'd personally want it to work just like close() so it closes with a valid POI even if it has to go back to failed block to do so.